### PR TITLE
Run lint on PRs

### PR DESCRIPTION
**Pull request by cherewaty**
This was the same approach taken for `stripes-components` - I assume it applies here too.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 
 buildNPM {
   publishModDescriptor = 'yes'
-}  
+  runLint = 'yes'
```
Comment (cherewaty) [inline Jenkinsfile]: This didn't appear to have any effect - there must be some config in Jenkins that isn't part of the repo.

**Comment by funkymalc** [comment]
Looks like stripes-core is eslint error free.   I am curious what to make of the bugs and vulnerabilities reported by Sonarqube.  https://sonarcloud.io/dashboard?id=folio-org%3Astripes-core

**Comment by cherewaty** [comment]
I'd rather not have sonarqube running in addition to eslint. Most of those "Remove this useless assignment" errors are disabled in eslint. It can be debated that maybe those shouldn't be disabled (some of them are easy to fix), but it's going to be difficult to maintain more than one static analysis source long-term.

**Comment by MikeTaylor** [comment]
Agreed, though I don't feel strongly about it.

**Pull request merged**